.gitignore

```diff
@@ -6,6 +6,8 @@ __pycache__/
 .coverage
 htmlcov
 .vscode/
+.pytest_cache
```

Comment:
Not necessary to update for now, but these are not needed. These two folders have their own `.gitignore` inside them which exclude everything in them.